front/src/core-logic/domain/admin/icUsersAdmin/icUsersAdmin.selectors.ts

```diff
@@ -21,6 +21,11 @@ const agencyUsers = createSelector(
   ({ agencyUsers }) => agencyUsers,
 );
 
+const isUpdating = createSelector(
```

Comment:
peut être plus explicite le nommage, car pas mal de choses peuvent s'updater :
```suggestion
const isUpdatingIcUserAgency = createSelector(
```

Reply:
pris en compte